### PR TITLE
Prepare a 0.3.0 release.

### PR DESCRIPTION
**Pull request by ltratt**

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,3 +1,19 @@
+# snare 0.3.0 (2020-03-06)
+
+## Breaking changes
+
+* `snare` now only searches for a configuration file at `/etc/snare/snare.conf`.
```
Comment (vext01) [inline CHANGES.md]: Shall we add prose to talk about `-c`?

**Comment by vext01** [comment]
One small comment

File: CHANGES.md (hunk 1 above)
Comment (ltratt) [reply CHANGES.md]: Fixed in https://github.com/softdevteam/snare/pull/56/commits/a75528a12e3b67ebe62089490bcc421bd13180bf.

**Comment by vext01** [comment]
Please squash

**Comment by ltratt** [comment]
Squashed.

**Pull request merged**